src/shared/constants.js

```diff
@@ -1,2 +1,2 @@
-export const MIN_SEARCH_TEXT_LENGTH = 2
```

Comment:
This has implications for the performance of the picker. For instance, when typing `s`, it returns a lot of results and so it may be slow. When I test on my laptop with 6x CPU throttling in Chrome, it takes ~1.3s to return this query. Whereas with two letters (e.g. `sm`), it takes 450ms.

Since people have to type one letter before they can type 2, it may make the UI appear sluggish.

So I prefer keeping the default at 2, but if people really want 1, they can change it themselves using something like [patch-package](https://www.npmjs.com/package/patch-package).

Arguably we could make this a configuration option. That could be a footgun, since it affects both the database and the picker. But if people really want 1, maybe it's reasonable to make it configurable.